libhb/work.c

```diff
@@ -1448,7 +1452,18 @@ static void do_job(hb_job_t *job)
     w = hb_get_work(job->h, WORK_READER);
     hb_list_add(job->list_work, w);
 
-    hb_log( "starting job" );
+    if (job->indepth_scan)
+    {
+        hb_log( "Starting Task: Subtitle Scan" );
+    } 
+    else if (job->pass_id == HB_PASS_ENCODE_1ST) 
+    {
+        hb_log( "Starting Task: Encode First Pass" );
+    } 
+    else 
+    {
+        hb_log( "Starting Task: Encode Second Pass" );
```

Comment:
This appears a bit odd when there is no first video pass. Perhaps we should change the messages to "Analysis Pass" and "Encoding Pass". I also notice there appears to be no information about the foreign audio scan pass, but this can be added later.